sdk/fabric/Azure.ResourceManager.Fabric/samples/Generated/Samples/Sample_FabricCapacityCollection.cs

```diff
@@ -22,7 +22,7 @@ public partial class Sample_FabricCapacityCollection
         public async Task CreateOrUpdate_CreateOrUpdateACapacity()
         {
             // Generated from example definition: 2023-11-01/FabricCapacities_CreateOrUpdate.json
-            // this example is just showing the usage of "FabricCapacity_CreateOrUpdate" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "FabricCapacities_createOrUpdate" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `createOrUpdate`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/fabric/Microsoft.Fabric.Management/capacity.tsp#L34